core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -600,8 +600,13 @@ public Journaled getDelegate() {
 
   @Override
   public JournalContext createJournalContext() throws UnavailableException {
+    return createJournalContext(true);
```

Comment:
just thinking could you add more comments on what will be the implementation class when this is T/F? Like describe if false then MasterJournalContext will be used which has no merging, and if true then it's FileSystemMergeJournalContext.